### ISSUE TITLE
Spreadsheet::send not worked for Xlsx writer

### ISSUE THREAD

**Issue by heggi** [yii2tech/spreadsheet#11]
Code returns file with size 0 bytes for writerType Xlsx and works fine for xls.
```
$exporter = new Spreadsheet([
     'dataProvider' => new ActiveDataProvider([
            'query' => $this->remains($c, $s),
      ]),
      'writerType' => 'Xlsx',
]);
return $exporter->send('remains.xlsx');
```
Problem here
https://github.com/PHPOffice/PhpSpreadsheet/blob/a1e8c843b79c6bc916cd2a5a725fc26948daba66/src/PhpSpreadsheet/Writer/Xlsx.php#L212
Created temporary file unlink and create a new file with same name, but here https://github.com/yii2tech/spreadsheet/blob/1ef9564a471dd55c6df021cc464af4f31ead4f67/src/Spreadsheet.php#L720 
sendStreamAsFile send old resource.

**Comment by klimov-paul** [comment]
Resolved by commit 356a769a229611b9e54c277a98b5fe29b6c75051

**Comment by mailgagannow** [comment]
Hi i am using your yii2 spreadsheet extension and I am trying to generate excel sheet via pagination but why it generating the excel sheet for all records. For example I have 20 records and my pagesize is 5 it still generating the excel sheet with all 20 records
